app/classes/box.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
@pellaea Would you mind quickly eyeballing this small class?
(Strangely enough, it's the first pure Ruby class I've written. 
I don't know how I managed to avoid that for so long.)

Reply:
There rarely is a reason to... Rails has pre-empted just about every class you could possibly want. :)